movie/src/components/Navbar/Navbar.css

```diff
@@ -0,0 +1,57 @@
+.navbar {
+  width: 70%;
+  margin: auto;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+}
+
+.left-nav {
+  display: flex;
+  justify-content: start;
+  align-items: flex-start;
+}
+
+.right-nav {
+  display: flex;
+  justify-content: end;
+  align-items: flex-start;
+}
+
+li {
+  text-decoration: none;
+  list-style: none;
+  color: white;
+  cursor: pointer;
+}
+.li-each-nav {
+  color: rgb(27, 24, 24);
+}
+
+.li-nav {
+  color: white;
+  margin: 15px;
+  font-weight: bold;
+}
+
+.list-li {
+  border-radius: 10px;
+  border: 1px solid rgb(205, 201, 201);
+}
+
+.logo-container {
+  display: flex;
+}
+
+.logo {
+  display: blox;
+  margin: 15px 15px 15px 0px;
+  font-weight: bolder;
+}
+.logo-color {
```

Comment:
Is the logo color going to be static or dynamic? if its a static color I think we don't need both `.logo` and `.logo-color`